gnucash/gtkbuilder/assistant-stock-transaction.glade

```diff
@@ -0,0 +1,941 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Generated with glade 3.38.2 -->
+<interface>
+  <requires lib="gtk+" version="3.22"/>
+  <object class="GtkAssistant" id="stock_transaction_assistant">
+    <property name="can-focus">False</property>
+    <property name="halign">start</property>
+    <property name="border-width">12</property>
+    <property name="title" translatable="yes">Stock Split Assistant</property>
+    <property name="window-position">center</property>
+    <property name="default-width">500</property>
+    <signal name="cancel" handler="stock_assistant_cancel" swapped="no"/>
+    <signal name="close" handler="stock_assistant_finish" swapped="no"/>
+    <signal name="prepare" handler="stock_assistant_prepare" swapped="no"/>
+    <child>
+      <object class="GtkLabel" id="intro_page_label">
+        <property name="visible">True</property>
+        <property name="can-focus">False</property>
+        <property name="valign">start</property>
+        <property name="margin-start">12</property>
+        <property name="margin-end">12</property>
+        <property name="label" translatable="yes">This assistant will help you record a stock transaction. The transaction type (purchase, sale, dividend, distribution, return of capital, stock split) will determine the transaction splits involved in the transaction.</property>
```

Comment:
Is the meaning of `distribution`somewhere explained, perhaps in the glossary?

Reply:
Another, more fundamental question: Is it only applicable for shares? Some other commodities like funds have similar tramsactions.

Reply:
A "distribution" in this meaning is a transfer of a company's retained wealth to the shareholders/partners. Dividends are one form of distribution used by regular companies and regulated investment companies (often known colloquially as open-end mutual funds). Other corporate structures, e.g. master limited partnerships, may call their distributions something else and they may have different tax implications.